pkg/vm/evm_prompts.go

```diff
@@ -877,51 +877,41 @@ func promptPermissioning(
 	return params, nil
 }
 
-func PromptVMVersion(
+func PromptSubnetEVMVersion(
 	app *application.Avalanche,
-	repoName string,
-	vmVersion string,
+	subnetEVMVersion string,
 ) (string, error) {
-	switch vmVersion {
+	switch subnetEVMVersion {
 	case latest:
-		return app.Downloader.GetLatestReleaseVersion(
-			constants.AvaLabsOrg,
-			repoName,
-			"",
-		)
+		return GetLatestCLISupportedDependencyVersion(app, constants.SubnetEVMRepoName, models.UndefinedNetwork, nil)
 	case preRelease:
 		return app.Downloader.GetLatestPreReleaseVersion(
 			constants.AvaLabsOrg,
-			repoName,
+			constants.SubnetEVMRepoName,
 			"",
 		)
 	case "":
-		return promptUserForVMVersion(app, repoName)
+		return promptUserForSubnetEVMVersion(app)
 	}
-	return vmVersion, nil
+	return subnetEVMVersion, nil
 }
 
-func promptUserForVMVersion(
+func promptUserForSubnetEVMVersion(
```

Comment:
this function is used for subnet evm only, renaming it to reflect this

Reply:
GetLatestCLISupportedDependencyVersion is the generic version

Reply:
This PR currently just looks at current implementation of how we handle default versions of subnet evm and avalanchego version and replace all current implementation of `Getlatestversion` for each dependency to `GetLatestCLISupportedDependencyVersion`